document/CoreFormat.mdk

```diff
@@ -805,83 +903,91 @@ If present, the `kind` property shall have one of the following values, with the
 * `notApplicable`: The rule specified by the `ruleId` property was not evaluated, because it does not apply to the
 file specified by `analysisTarget` (&sect;[#location-analysisTarget]).
 
-~ IsoExample
-A binary checker might have a rule that applies to 32-bit binaries only.
-Such a tool would produce a `notApplicable` result if it were run on a 64-bit binary.
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+In this example, a binary checker has a rule that applies to 32-bit binaries only.
+It produces a `notApplicable` result if it is run on a 64-bit binary:
+```
+    "results": [
+        {
+            "ruleId": "ABC0001",
+            "level": "notApplicable",
+            "message": "\"MyTool64.exe\" was not evaluated for rule ABC0001 because it is not a 32-bit binary."
+            "locations": [
+                {
+                    "analysisTarget": {
+                        "uri": "file://C:/bin/MyTool64.exe"
+                    }
+                }
+            ]
+        }
+    ]
+```
 ~
 
-* `configurationError`: The tool encountered a rule configuration error.
-
-    The `ruleId` property for a `result` object whose `kind` property is `configurationError` should be
-    set to a unique value, distinct from the id of all other rules known to that tool, which indicates that
-    an error occurred while reading rule configuration information.
+* `note`: A purely informational log entry.
 
-~ IsoNote
-Many tools can be parameterized with information about which rules to run, and how they should be configured.
-In some cases, if the configuration information is invalid, the tool can ignore the invalid information
-and continue to run.
-~
+   The `ruleId` property for a `result` object whose `kind` property is `note` may be present, if the
+   note relates to a particular rule; otherwise `ruleId` may be absent.
 
-~ IsoExample
-Suppose a tool is invoked with a configuration file which specifies that the tool should disable rule X.
-If there is no such rule as rule X, then the tool should report a `configurationError`. The tool might
-continue to run, reporting results for the rules that are correctly configured.
+~ IsoNumberedExample
+In this example, the tool reports an observation about the code that does not
+represent a problem.
+```
+    "results": [
+        {
+            "ruleId": "ABC0002",
+            "level": "note",
+            "message": "Consider using 'nameof(start)' instead of hard-coding the parameter name 'start'."
```

Comment:
I continue to see this as a warning. This isn't purely information, you're telling the developer about a potential problem: they may rename parameter 'start' and neglected to update the hard-coded string literal referring to it. Using nameof will eliminate this potential maintainability problem. Good result. Not an 'info'. 'info' object provide observational information. this information may _support_ analysis or a result. but the item itself shouldn't constitute a discrete issue which you might want to track and fix. our experience says that three levels of priority for things you might fix is too much: the p3s are just noise. developers put things into two buckets: must fix, nice to fix. errs & warnings, iow.